Writing_Plugins.md

```diff
@@ -12,6 +12,8 @@ Try and keep your plugins as cross-framework compatible as possible. Here are so
 
 5. If you’re making a theme, please include a screenshot so prospective users can see what it looks like without having to install it.
 
+6. If your plugin adds any of its subdirectories to the `fpath`, make sure those subdirectories only contain function definition files. This allows for frameworks to correctly [zcompile all functions](http://zsh.sourceforge.net/Doc/Release/Functions.html#Autoloading-Functions). Don't make you plugin add its root directly to the `fpath`.
```

Comment:
One minor typo fix `s/make you plugin/make your plugin/` and it is good to go

Reply:
Done

Reply:
Also did `s/root directly/root directory` which was a typo that even sounded correct at first.  :- )